ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -35,15 +35,14 @@ export default function AdvancedGasControls({
   maxPriorityFeeFiat,
   maxFeeFiat,
   gasErrors,
-  networkSupportsEIP1559,
   minimumGasLimit,
 }) {
   const t = useContext(I18nContext);
-  const networkSupports1559 = useSelector(isEIP1559Network);
+  const use1559 = useSelector(networkAndAccountSupports1559);
```

Comment:
nit: do we want to introduce some naming convention for variables and functions that return boolean values?
E.g. `const networkAndAccountSupport1559 = useSelector(doesNetworkAndAccountSupport1559)`. In general, something like this: `variableName = doesVariableName() | isVariableName() | hasVariableName()`.